catboost/libs/metrics/stress-test/fairloss_stress_text.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+from catboost.utils import eval_metric
+
+def fairloss(actual, pred):
+    x = abs(np.array(actual) - np.array(pred))
+    c = 2
+    res = np.mean(c**2 * ((x / c) - np.log((x/c) + 1)))
+    return res
+
+list_sizes = [10**exp for exp in range(1, 7)]
+
+for size in list_sizes:
+    print("Evaluating for dataset size =", size)
+    for k in range(100):
+        actual = np.random.randint(0, 2, size)
```

Comment:
you test is not deterministic, you should use `np.random.RandomState` with fixed `seed` for random number generation.